UnitTests/BraintreePayPalTests/BTPayPalRequest_Tests.swift

```diff
@@ -46,8 +46,12 @@ class BTPayPalRequest_Tests: XCTestCase {
         request.riskCorrelationID = "123-correlation-id"
         request.merchantAccountID = "merchant-account-id"
         request.isShippingAddressEditable = true
-
-        request.lineItems = [BTPayPalLineItem(quantity: "1", unitAmount: "1", name: "item", kind: .credit)]
```

Comment:
Can we keep the test for the original .credit kind and create another test for the .debit kind?